eth/api.go

```diff
@@ -54,6 +54,9 @@ import (
 
 const defaultGas = uint64(90000)
 
+// TODO(tzdybal) - make it configurable parameter
+const replayTransactions = true
```

Comment:
w/r/t [my later comment](https://github.com/ethereumproject/go-ethereum/pull/634/files#r195641141), is this intended in order to make the API `eth_getTransactionReceipt` output configurable? Or?

Reply:
Yes I was thinking about making it an option. Because, when we add dynamic state fetching, we will have a complicated situation for "fast" clients. After querying transactions, client won't be typically fast, the extra state will require extra disk space. And generation of status by re-executing transactions is extra cost (and maybe someone doesn't need status field).